.eslintrc

```diff
@@ -1,5 +1,8 @@
 {
     "parser": "babel-eslint",
+    "extends": [
+      "defaults/configurations/google"
```

Comment:
mmm you're right, we did use gjslint (we don't for a long time now but that's actually our defaults).

I'll have a deeper look on the rules then.